sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -93,6 +93,14 @@ def create_experiment(
 
         namespace = namespace or self.namespace
 
+        if experiment.metadata.name in [
+            experiment.metadata.name
+            for experiment in self.list_experiments(namespace=namespace)
```

Comment:
Should [we use `get_experiment()`](https://github.com/kubeflow/katib/blob/bf1ce98a832c4f76ede1afe1b54fee0bac92723c/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L361) API instead ?

Reply:
The problem with `get_experiment` is that if the experiment doesn't exist, the API returns a 404, in which case we have to check for a 404 as well, at which point we might as well just check for a 409 in the create like we did originally, heh.